lighthouse-cli/test/smokehouse/seo/expectations.js

```diff
@@ -213,6 +214,7 @@ module.exports = [
                   '\n        big enough target\n      </a>',
                   'path': '2,HTML,1,BODY,3,DIV,21,DIV,1,A',
                   'selector': 'body > div > div > a',
```

Comment:
can you add to the `a11y` expectations as well?

Reply:
Right now all we assert there is `{ length: 1 }`, but I'll change it when I do the final artifact update for this PR.

Reply:
Added `details.items` assertions.

Reply:
> Right now all we assert there is { length: 1 }, but I'll change it when I do the final artifact update for this PR.

haha, uh oh, I guess I didn't know how much I was asking for :) I guess we can just cross our fingers that these are stable and all worth testing. I do really like the extra coverage this gets us

Reply:
When @mattzeunert does something, he does it right, and you get the works! 😎